uber/models.py

```diff
@@ -1938,6 +1953,22 @@ class ArbitraryCharge(MagModel):
     _repr_attr_names = ['what']
 
 
+class LedgerItem(MagModel):
+    ident = Column(UnicodeText, nullable=True)
```

Comment:
I'm not sure what purpose this ``ident`` serves - what's an example of an ident value for a row in this table and how it would be chosen.